### ISSUE TITLE
Running doctor returns error "Cannot read property 'filter' of null"

### ISSUE THREAD

**Issue by nevardreik** [NativeScript/nativescript-cli#5366]
**Environment**
 - CLI: 7.0.4
 - Android Runtime: 7.0.0
 - iOS Runtime: 7.0.0

**Describe the bug**
When running `ns doctor` the following error occurs.

```
 nevardreik@dev.local 💀  > ~/develop/mobile > ns doctor
✖ Getting environment information 

Cannot read property 'filter' of null
```

**To Reproduce**

**Expected behavior**
Command should run successfully and advise why my app is probably not working 😅

**Comment by rigor789** [comment]
Can you share the folder structure of `~/develop/mobile`?

**Comment by nevardreik** [comment]
Sure - at a high level:

```
mobile
|- app
  |- App_Resources
  |- main.ts
  |- app.module.ts
  |- app.component.ts
  |- app.component.html
  |- app.scss
|- nativescript.config.ts
|- package.json
|- package-lock.json
|- tsconfig.json
|- tsconfig.esm.json
|- tsconfig.tns.json
|- webpack.config.json
--

**Comment by rigor789** [comment]
Ok, that looks normal - contents of `package.json`? Perhaps if you could share the project (you can send the zip to igor@nstudio.io) I could debug it easier!

**Comment by nevardreik** [comment]
Hey Igor, I've sent you an email with details to sample project. Thanks!

**Comment by rigor789** [comment]
@nevardreik Thanks - I was able to track down the issue, and released a fix to `@nativescript/doctor` - the main cause was the `~7.0.0` version in the `package.json` for the android runtime - `semver.valid` throws on ranges - but that should be fixed now.

The latest master with this change has been auto-released as `7.0.6-2020-09-06-123725-14246` but I will be releasing a patch version today/tomorrow with other fixes I do.

**Comment by nevardreik** [comment]
Nice one, thank you @rigor789!
